airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/manifest_only_connectors.py

```diff
@@ -46,3 +54,19 @@ def get_test_steps(context: ConnectorTestContext) -> STEP_TREE:
             )
         ],
     ]
+
+
+class ManifestOnlyConnectorUnitTests(PytestStep):
+    """A step to run unit tests for a manifest-only connector"""
+
+    title = "Manifest-only unit tests"
+    test_directory_name = "unit_tests"
+
+    # No coverage reporting needed since we're just testing components.py
+    common_test_dependencies = ["pytest"]
+
+    @property
+    def default_params(self) -> STEP_PARAMS:
+        """Default pytest options without coverage computation"""
+
+        return super().default_params
```

Comment:
❗ This shouldnt be nessesary right? What weve written here is the same behaviour as if this method wasn't here
```suggestion

```

Reply:
Yup, who knows what I was thinking here? 😅